domains/emissions/Emissions.API/API/Services/EmissionsCalculator.cs

```diff
@@ -6,10 +6,14 @@ namespace API.Services;
 class EmissionsCalculator : IEmissionsCalculator
 {
 
-    private const int decimalPrecision = 5;
-
-    public EmissionsResponse CalculateEmission(IEnumerable<EmissionRecord> emissions,
-        IEnumerable<TimeSeries> measurements, long dateFrom, long dateTo, Aggregation aggregation)
+    const int decimalPrecision = 5;

```

Comment:
Should we extract this to some configuration, it exists in both calculators..

Reply:
As in, a configurable value (environment variable)?
Or just as a single constant within the solution?

Reply:
One constant for now, then we can always move it to an environment variable later, unless the change is minor to do so, and set a default variable to 5.

Reply:
Moved to common static property.